code/game/objects/effects/portals.dm

```diff
@@ -185,74 +186,43 @@
 /obj/effect/portal/permanent
 	name = "permanent portal"
 	desc = "An unwavering portal that will never fade."
-	var/id // var edit or set id in map editor
 	hardlinked = FALSE // dont qdel my portal nerd
+	force_teleport = TRUE // force teleports because they're a mapmaker tool
+	var/id // var edit or set id in map editor
 
-/obj/effect/portal/permanent/Initialize(mapload, _lifespan = 0, obj/effect/portal/_linked, automatic_link = FALSE, turf/hard_target_override, atmos_link_override)

```

Comment:
was this intended to remove initialize?

Reply:
yes it re-checks links on every teleport now